src/subcommand/server.rs

```diff
@@ -466,7 +465,7 @@ impl Server {
   }
 
   async fn sat(
-    Extension(page_config): Extension<Arc<PageConfig>>,
+    Extension(page_config): Extension<Arc<ServerConfig>>,
```

Comment:
We should change `page_config` to server config:

```suggestion
    Extension(server_config): Extension<Arc<ServerConfig>>,
```

There's also a bunch of other places we call it `page_config`, the easiest way to find them is probably just grepping.